tests/rptest/e2e_tests/workloads/flink_table_transactions_basic.py

```diff
@@ -101,6 +102,10 @@ def setup(self):
 
         # Alternative way to add connector, just for illustrative purposes
         table_env.get_config().set("pipeline.jars", self.config.connector_path)
+        # This is needed for Scalar functions work
+        # See https://nightlies.apache.org/flink/flink-docs-master/docs/dev/python/dependency_management/
+        table_env.get_config().set("pipeline.classpaths",
```

Comment:
This is critical due to Classpath loader can't find Python functions and this jar is located outside lib folder